sites/gcweb-menu/gcweb-menu-fr.html

```diff
@@ -0,0 +1,44 @@
+---
+{
+	"title": "Menu GCWeb",
+	"language": "fr",
+	"altLangPage": "gcweb-menu-en.html",
+	"secondlevel": false,
+	"dateModified": "2022-05-10",
+	"share": "true"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>La <a href="#v1">première version (v1) de menu</a> est obsolète mais toujours supportée.</p>
+
+<h2>Version 2</h2>
```

Comment:
Je décrirerais le changement majeur. par example:
```suggestion
<h2>Version 2</h2>
<p lang="en">Need to be used along with the <a href="../header/header-en.html">version 2 of site header</a></p>
<p lang="en">List of change since version 1: Removal of the <code>&lt;div class="container"&gt;</code>.</p>
```